src/bench/examples.cpp

```diff
@@ -5,29 +5,18 @@
 #include <bench/bench.h>
 #include <util/time.h>
 
-// Sanity test: this should loop ten times, and
-// min/max/average should be close to 100ms.
-static void Sleep100ms(benchmark::State& state)
```

Comment:
Why did you remove this test? (should probably move to tests though)

Reply:
It seems to have been added here: https://github.com/bitcoin/bitcoin/commit/535ed9223dcb32bf90ead5b2c95052838b780620#diff-5f8387aba8e5e6c0c871e093c9145085R9
So I guess it is somewhat a useless test

Reply:
I removed this test because it's rather useless, it would uselessly slow down the whole benchmark run quite a bit compared to the others, and since it's in `examples.cpp` I assumed that it's just an example anyways

Reply:
Unless I was missing something, if you remove this test then the `#include <util/time.h>` can be removed with it.

Reply:
Ah right I totally forgot removing the include

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/18011/commits/5bd582bbadb90970f631a7c4c4d793689584ca4e